src/plugins/workspace/common/constants.ts

```diff
@@ -3,6 +3,13 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+export const WORKSPACE_CREATE_APP_ID = 'workspace_create';
+export const WORKSPACE_LIST_APP_ID = 'workspace_list';
+export const WORKSPACE_UPDATE_APP_ID = 'workspace_update';
+export const WORKSPACE_OVERVIEW_APP_ID = 'workspace_overview';
+// These features will be checked and disabled in checkbox on default.
```

Comment:
Can you give more detail about comments here? I'm a little bit lost here based on the comment here. thanks

Reply:
The user needs to select which features will be enabled when creating or updating workspace. The enabled features will be list on the left navigation. The workspace overview and workspace update should be selected by default which means every workspace should have these two pages.